tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/OpenNewConnectionWizardTest.java

```diff
@@ -28,50 +27,44 @@
  * Test opening new connection dialog via tool item and context menu.
  * 
  * @author mlabuda@redhat.com
+ * @author adietish@redhat.com
  *
  */
-public class OpenNewConnectionShellTest {
+public class OpenNewConnectionWizardTest {
 	
 	OpenShiftExplorerView explorer = new OpenShiftExplorerView();
 	
 	@Test
-	public void testOpenConnectionShellViaLinkTest() {
+	public void shouldOpenConnectionWizardViaLinkTest() {
+		// given
+		ConnectionUtils.clearAll();
 		explorer.open();
-		
-		try {
-			new DefaultLink(OpenShiftLabel.TextLabels.NEW_CONNECTION).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via context menu.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+
+		// when
+		new DefaultLink(OpenShiftLabel.TextLabels.CREATE_CONNECTION).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
 	@Test
-	public void testOpenConnectionShellViaToolItemTest() {
+	public void shouldOpenConnectionWizardViaToolItemTest() {
+		// given
 		explorer.open();
 		
-		try {
-			new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via view tool item.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+		// when
+		new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
-	private void verifyNewConnectionShellIsOpened() {
-		Shell connectionShell = null;
-		try {
-			connectionShell = new DefaultShell(OpenShiftLabel.Shell.NEW_CONNECTION);
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell was not opened.");
-		}
-		
+	private void verifyNewConnectionWizardIsOpened() {
```

Comment:
You have different result if you run it. E.g. you run it from maven, you get test result and there are failed tests and tests in error. You can expect failed tests and have meaningful output. Well you want output in form of passed and failed tests. But you dont know why there are errors for some tests and you need to examine further. Me and Radim just talked about this and we agreed that the best approach would be attaching stacktrace to output. E.g. ex.printStackTrace(); before fail. Or we could create a method similar to https://paste.fedoraproject.org/488495/98266161/ and use it to print stack trace after failure message.